documentation/deliberately_vulnerable_flask_app/README.md

```diff
@@ -3,6 +3,6 @@ This is a trivially vulnerable Flask application, meant as a playground for
 testing Pysa
 
 # Setup
-1. `pip install pyre-check-nightly`
+1. `pip install pyre-check-nightly flask lxml`
```

Comment:
This may conflict with https://github.com/facebook/pyre-check/pull/438 in which we refactored this to use a requirements.txt . No action needed now but if we merge https://github.com/facebook/pyre-check/pull/438 before you should update the `requirements.txt` file before